README.md

```diff
@@ -1,2 +1,37 @@
 # Building-Blocks-Enumerables
-Rebuild Enumerables Methods
+The objective of this project is to rebuild Enumerable methods and achieve the same results as the originals.
+
+## Built With
+
+- Ruby 2.6.5
+
+## Getting Started
+
+If you have Ruby installed, you can use the next command to run the code: ruby main.rb
+
```

Comment:
That's ok, but you can't play with the methods like that. You could give instructions to open the ruby console and test your methods there.